http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -276,6 +276,14 @@ STEP_TIMINGS+=($(date +%s))
 
 log 'Starting up the Jupyter...'
 
+
+# Start gcsfuse as a sidecar
+mkdir -p /mnt/disks/bucket
+docker run -d --name gcsfuse-container --privileged  -u root -e PIP_USER=false  --env BUCKET_NAME=genomics-public-data   --device /dev/fuse:/dev/fuse   --security-opt apparmor=unconfined   -v /mnt/disks/bucket:/mnt/gcs-bucket:shared   tarekmahmed/gcsfuse-container:latest
```

Comment:
FYI, you likely want to use the --restart always option so that the volume get's mounted properly after pausing / resuming your runtime, see https://github.com/DataBiosphere/leonardo/blob/a9a9c0f683445f1c7dbda3543e80ee7bc35e6d9a/http/src/main/resources/init-resources/azure_vm_init_script.sh#L240